src/xo-app/sr/tab-disks.js

```diff
@@ -74,14 +74,19 @@ const COLUMNS = [
   }
 ]
 
+const FILTERS = {
+  filterOnlySnapshots: 'snapshot',
+  filterNoSnapshots: '!snapshot'
```

Comment:
Use the more accurate filters to avoid false-positives:

```
filterNoSnapshots: 'type:!VDI-snapshot',
filterOnlySnapshots: 'type:VDI-snapshot'
```